documentation/puppet_server_metrics_performance.markdown

```diff
@@ -31,6 +31,8 @@ Puppet Server uses JRuby, which rations server resources in the form of JRubies
 
 Several factors can limit your Server infrastructure's ability to produce JRubies.
 
+> **Note:** In multithreaded mode, instead of producing multiple independent JRuby instances, the server runs a single JRuby instance that can process a limited number of threads. These two concepts are analogous when considering request-handling capacity, though threads are much cheaper in terms of system resources, so you may have more vertical scalability in multithreaded mode.
```

Comment:
This note seems a little out of place here? The first sentence seems relevant to mention here, but the second should maybe go in the `Request-handling capacity` section?

Reply:
I can split it up.